app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
I think the using regular args is fair enough here. You're right Ben that it causes 'duplication' when you're passing `content_item_id: content_item.id`. It's in the tests where it is more readable but that causes uglification™ in 'real life'. 

As for the `increment`/`next`... I actually had it called `next` originally but it felt a bit passive. Like a 'get' kind of thing. `increment` felt more descriptive in that that's what it actually does. The fact that it returns the value is almost a byproduct (and arguably unnecessary now the use of it has fleshed out a bit)